spec/lib/mavenlink/client_spec.rb

```diff
@@ -13,6 +13,46 @@
     end
   end
 
+  describe 'association calls' do
+    subject(:client) { described_class.new(oauth_token: '12345') }
+    let(:record) { client.workspaces.find(7) }
+
+    let(:response) {
+      {
+        'count' => 1,
+        'results' => [{'key' => 'workspaces', 'id' => '7'}, {'key' => 'users', 'id' => '2'}],
+        'users' => {
+          '2' => {
+            'id' => 2,
+            'full_name' => 'John Doe'
+          }
+        },
+        'workspaces' => {
+          '7' => {
+            'title' => 'My new project', 'id' => '7',
+            'participant_ids' => ['2'],
+          }
+        }
+      }
+    }
+
+    before do
+      stub_request :get, '/api/v1/workspaces?only=7', response
+    end
+
+    specify do
+      expect(record.participants.count).to eq(1)
+    end
+
+    specify do
+      expect(record.participants.first).to be_a(Mavenlink::User)
+    end
+
+    it 'saves the client scope' do
+      expect(record.participants.first.client).to eq(client)
```

Comment:
@cantino this is the case: http://stackoverflow.com/questions/29288253/ruby-instance-variable-across-classes-or-a-better-way-to-share-an-object-among-c